var/spack/repos/builtin/packages/amg2013/package.py

```diff
@@ -40,24 +40,28 @@ class Amg2013(MakefilePackage):
             url='https://codesign.llnl.gov/amg2013/amg2013.tgz')
 
     variant('openmp', default=True, description='Build with OpenMP support')
-    variant('assumepartition', default=False, description='Assumed partition (for thousands of processors)')
+    variant('assumedpartition', default=False, description='Use assumed partition (for thousands of processors)')
+    variant('64bitglobals', default=False, description='Use 64-bit integers for global variables')
```

Comment:
please call it `int64` to be consistent with other packages (superlu-dist, mumps, hypre, petsc, etc)